### ISSUE TITLE
Can output be exported to standard formats like JUnit?

### ISSUE THREAD

**Issue by ghost** [rpm-software-management/rpmlint#109]
Can rpmlint output be exported to standard formats like JUnit?

**Comment by dirkmueller** [comment]
That's more of a question than an issue, isn't it?

I think there is currently no support for that.. do you need it? What would be the use case?

**Comment by pantocrator27** [comment]
It is :)

The use case is for running rpmlint on a CI server like Jenkins and using
the standard reporting for reporting using Jenkins.

**Comment by scop** [comment]
Right, currently no support for that exists, and none is planned AFAIK. If someone wants to take a stab at it, here's py.test's --junitxml option related code: https://github.com/pytest-dev/pytest/blob/master/_pytest/junitxml.py (although I think a simple separate shell pipe wrapper script  would cover most if not all of the test cases).

**Comment by brianjmurrell** [comment]
[CS]ouldn't this issue stay open as an RFE to have JUnit support?  Then somebody could come along looking for open issues to work on and perhaps pick this one up.
